### DISCUSSION TITLE
<title>repository

### DISCUSSION THREAD

**Discussion by Dev-Anoop0000** [community/community#48639] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My repository not run on git hub website .
i am unable to download any file on github.
my friend search me on github but he not found my id.
There is something problem in my account please help me.

**Comment by DRUMPADD** [comment]
Hello. I could find you 😃. Maybe your friend make a mistake trying to find you. Did you already create your repository on github?

**Comment by Dev-Anoop0000** [comment]
No no no.....
That's problem you mentioned , it's last problem. My repository file 📁
does not render on display. Means file not open and does not show.
Then tell me.
How I public my repository.
Please try to understand my problem.

**Comment by DRUMPADD** [comment]
Sorry, you can check this: https://docs.github.com/en/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-locally-hosted-code-to-github. I hope this is be really helpful
